vaadin-grid-pro-flow/src/main/resources/META-INF/resources/frontend/editColumnConnector.js

```diff
@@ -0,0 +1,18 @@
+window.Vaadin.Flow.editColumnConnector = {
```

Comment:
Actually, I would prefer to call this file and class `gridProConnector`. This way, if we need more client-side customization for `GridPro` in the future, we don't have to create a new connector, and the name is still semantically correct and not misleading.

Reply:
Renamed.